lib/auth/grpcserver.go

```diff
@@ -39,6 +39,7 @@ import (
 	"google.golang.org/grpc/keepalive"
 	"google.golang.org/grpc/peer"
 	"google.golang.org/grpc/status"
+	gproto "google.golang.org/protobuf/proto"
```

Comment:
Is the alias necessary? I'd rather alias Gogo to gogoproto and keep "proto" for the Google one.

Now, if you really want to be unambiguous:

```suggestion
	googleproto "google.golang.org/protobuf/proto"
```

Reply:
I'll make this `googleproto`. The issue isn't `gogoproto` taking the package name, but `api/client/proto` within our codebase, unfortunately.

Reply:
Oh, right. I'd alias that one to `authpb` or something, then.